pkg/controllers/node/termination/node_claim_test.go

```diff
@@ -0,0 +1,537 @@
+/*
```

Comment:
We've been doing `nodeclaim` as one word for file naming so rename this to `nodeclaim_test.go`